renku/cli/dataset.py

```diff
@@ -217,6 +217,9 @@
 with those datasets. Modifying those datasets locally will prevent them from
 being updated.
 
+The update command also checks for files changes in the project and updates
+dataset's metadata accordingly.
```

Comment:
```suggestion
The update command also checks for file changes in the project and updates
datasets' metadata accordingly.
```

I think it's either `datasets' metadata` or `each dataset's metadata` if I'm not mistaken about english posessives (which I might well be, it's complicated). At least I think if you have a singular form it has to be determined by something like "the" or "each" or "this". Or maybe just `dataset metadata`? Not that it really matters, just "updates
dataset's metadata" sounds weird somehow.